modules/luci-base/root/etc/config/luci

```diff
@@ -24,7 +24,7 @@ config internal ccache
 config internal themes
 
 config internal apply
-	option rollback 30
```

Comment:
As we have discussed that Cu setup we are observing this issue as it's connected over vpn, So looks like it take more time sometime so previously if it didn't get response then it will rollback but with current changes rollback will not be there but we may see timeout warnning message so to avoid this increase this time. Openwrt prefer 120 s and initially we also had 120 s timeout then we reduce to 30s. But looks like for Cu scenario it's better to keep 60s.